rdtho2525/whats-cookin
# Feature/recipe class

## Is this a fix or a feature? ⬇️
+ Feature


## What is the change? ⬇️
+ Wrote test suite and implementation code for Recipe class


## What does it fix? ⬇️
+ n/a


## Where should the reviewer start? ⬇️
+ JS files Recipe-tests.js, Recipe.js in their entirety 


## How should this be tested? ⬇️
+ Compare test suite to required functionality in project spec/GH issues
+ Run `npm test tests/Recipe-tests.js` to ensure that the Recipe class passes each unique test
